### PR TITLE
Add support for Terraform v0.12

### PR DESCRIPTION
**Pull request by aaron-lane**

### CODE REVIEW

**Comment by Jberlinsky** [comment]
@aaron-lane Could you kindly rebase this against `master`? Thanks!

**Comment by aaron-lane** [comment]
@Jberlinsky rebase in progress!

File: README.md [hunk 1]
```diff
@@ -203,7 +202,7 @@ The [project factory](https://github.com/terraform-google-modules/terraform-goog
 #### Kubectl
 - [kubectl](https://github.com/kubernetes/kubernetes/releases) 1.9.x
 #### Terraform and Plugins
-- [Terraform](https://www.terraform.io/downloads.html) 0.11.x
+- [Terraform](https://www.terraform.io/downloads.html) 0.12
```
Comment (Jberlinsky) [inline README.md]: We should add a note like we did in https://github.com/terraform-google-modules/terraform-google-network/pull/55 pointing people to the right place for a 0.11.x-compatible version.

Comment (aaron-lane) [reply README.md]: Added in 6cc0bf8.

File: autogen/cluster_regional.tf [hunk 2]
```diff
@@ -20,78 +20,119 @@
...
     client_certificate_config {
-      issue_client_certificate = "${var.issue_client_certificate}"
+      issue_client_certificate = var.issue_client_certificate
     }
   }
 
   addons_config {
     http_load_balancing {
-      disabled = "${var.http_load_balancing ? 0 : 1}"
+      disabled = ! var.http_load_balancing
```
Comment (morgante) [inline autogen/cluster_regional.tf]: Not sure I love this approach but it's okay.

**Pull request merged**